src/DemoUI/src/Repository/SessionRepository.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Packlink\DemoUI\Repository;
+
+use Logeecom\Tests\Infrastructure\Common\TestComponents\ORM\MemoryRepository;
+
+/**
+ * Class SessionRepository.
+ *
+ * @package Packlink\DemoUI\Repository
+ */
+class SessionRepository extends MemoryRepository
+{
+    /**
+     * Fully qualified name of this class.
+     */
+    const THIS_CLASS_NAME = __CLASS__;
+
+    /**
+     * @inheritDoc
+     */
+    protected function getStorage()
+    {
+        $this->ensureStorage();
+
+        return $_SESSION['storage'];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function saveToStorage($key, $item)
```

Comment:
Codacy found an issue: [saveToStorage accesses the super-global variable $_SESSION.](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
Please ignore such things in the DemoUI folder, since this will not be used in any module.